app/assets/javascripts/controllers/editable_list_controller.js

```diff
@@ -0,0 +1,104 @@
+import { Controller } from "@hotwired/stimulus";
+
+function swap(target, item) {
+  if (target && target !== item) {
+    const positionComparison = target.compareDocumentPosition(item);
+    if (positionComparison & 4) {
+      target.insertAdjacentElement('beforebegin', item);
+    } else if (positionComparison & 2) {
```

Comment:
constants or documentation reference for the compare position magic numbers? e.g. `https://developer.mozilla.org/en-US/docs/Web/API/Node/compareDocumentPosition`